ReleaseTooling/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -756,28 +765,17 @@ struct ZipBuilder {
     for framework in podInfo.binaryFrameworks {
       // Copy it to the temporary directory and save it to our list of frameworks.
       let zipLocation = binaryZipDir.appendingPathComponent(framework.lastPathComponent)
-      let carthageLocation =
-        binaryCarthageDir.appendingPathComponent(framework.lastPathComponent)
 
       // Remove the framework if it exists since it could be out of date.
       fileManager.removeIfExists(at: zipLocation)
-      fileManager.removeIfExists(at: carthageLocation)
       do {
         try fileManager.copyItem(at: framework, to: zipLocation)
-        try fileManager.copyItem(at: framework, to: carthageLocation)
       } catch {
         fatalError("Cannot copy framework at \(framework) while " +
           "attempting to generate frameworks. \(error)")
       }
       frameworks.append(zipLocation)
-
-      CarthageUtils.generatePlistContents(
```

Comment:
Is this not relevant anymore?

Reply:
It's done in the xcframework build.  Ideally `generatePlistContents` should be moved out of CarthageUtils.  I can do that in a subsequent PR.